sway-parse/src/item/item_impl.rs

```diff
@@ -88,12 +88,28 @@ mod tests {
 
     #[test]
     fn parse_impl_slice() {
-        let item = parse::<ItemImpl>(
-            r#"
-            impl __slice[T] {}
-            "#,
+        // deprecated syntax
```

Comment:
Can you open a new issue for this and mention its number in a TODO here?

Reply:
Done. I put the todo near the parser, which I think makes more sense.